### ISSUE TITLE
Did you use some tricks to obtain better accuracy than libffm?

### ISSUE THREAD

**Issue by tcsere07** [alno/batch-learn#3]
Hey @alno , did you use some special tricks to obtain better accuracy than libffm?

I have to say it's really a amazing tool, thanks a lot.

**Comment by alno** [comment]
Hi, the only accuracy-related trick i've used is dropout - it reduced overfitting and improved final results.

**Comment by tcsere07** [comment]
Hi @alno , Thx for your reply. I used the tool to complete my task and I found that it converged faster than libffm, but it seems that you all used openmp to accumulate the training process, so what's the difference between them? And I also found that Even if I set the same seed, but still got different loss, Why?
